crmsh/lock.py

```diff
@@ -30,7 +30,7 @@ class Lock(object):
 
     LOCK_DIR = "/tmp/.crmsh_lock_directory"
     MKDIR_CMD = "mkdir {}".format(LOCK_DIR)
-    RM_CMD = "rm -rf {}".format(LOCK_DIR)
+    RM_CMD = "rm -rf {};sync".format(LOCK_DIR)
```

Comment:
Two comments,
1. sync is the system wide, likely overkill the whole system. I would avoid using it.
2. /run is gurenteed to be cleaned per boot, and fits your finding well. Seems, /run/crm is created by some one. If it is created by crmsh.rpm, then, it's safe to use, 
LOCK_DIR = "/run/crm/.crmsh_lock_directory"

Reply:
Good suggestions! Already tried to use `/run/crm/.crmsh_lock_directory`, seems work

Reply:
@zzhou1 `/run/crm` is created just when pacemaker service starting, not related with crmsh

Reply:
> @zzhou1 `/run/crm` is created just when pacemaker service starting, not related with crmsh

Then, it is is safe for crmsh to use it directly.